packages/core/src/renderer/Equation.ts

```diff
@@ -52,20 +53,20 @@ const Equation = ({ shape, canvasSize, labels }: ShapeProps): SVGGElement => {
   if (!labelFound) {
     // Fallback case: generate plain-text (non-rendered) label from string
     const txt = document.createElementNS("http://www.w3.org/2000/svg", "text");
-    txt.textContent = getAdValueAsString(shape.properties.string);
+    txt.textContent = getAdValueAsString(shape.string);
     attrToNotAutoMap.push("string");
     elem.appendChild(txt);
 
     // Map the attributes we have
     attrToNotAutoMap.push(...attrFill(shape, elem));
     attrToNotAutoMap.push(...attrWH(shape, elem));
-    attrToNotAutoMap.push(...attrStroke(shape, elem));
-    attrToNotAutoMap.push(...attrFont(shape, elem));
+    // attrToNotAutoMap.push(...attrStroke(shape, elem));
+    // attrToNotAutoMap.push(...attrFont(shape, elem));
```

Comment:
Probably delete these lines instead of commenting out?
```suggestion
```

Reply:
Yes